src/logstatsexp.jl

```diff
@@ -0,0 +1,38 @@
+"""
+    logmeanexp(A; dims=:)
+
+Computes `log.(mean(exp.(A); dims))`, in a numerically stable way.
+"""
+function logmeanexp(A::AbstractArray; dims=:)
+    R = logsumexp(A; dims)
```

Comment:
`logsumexp(A; dims)`, that syntax was not available in 1.0 I think, try using `dims=dims`